.github/workflows/queries.yml

```diff
@@ -48,6 +32,8 @@ jobs:
           git config --global url."https://github.com/".insteadOf
           ssh://git@github.com/
       - run: npm ci
+      - run: docker-compose up -d
+      - run: ./wait-for-it.sh localhost:7687 --timeout=80
```

Comment:
do you need this since you have configured the health-check already?

Reply:
We do, the healthcheck will only set the status on the container (you can see it using `docker stats`) but apparently  the command `docker-compose` does not wait for the container to become healthy.

The `docker-compose` command will start the container and return 0.

Reply:
oh yes, the service container defined in the GitHub Actions pipeline automatically waits for it, but if you move away from it (by using Docker Compose e.g.), you have to wait for the container yourself.
My next question then becomes: do you need docker-compose here? :)

Reply:
As far as I know, it's not possible to run a step/task before the GitHub Actions `services`. In our case, we need to enable several plugins which are not available as part of the Docker image such as `spatial.jar` and `apoc-nlp-dependencies-4.3.0.0.jar`. In other words, we need to mount a volume but since GitHub Actions `services` runs before `git clone`, it won't work.

I'm not entirely satisfied, so if you have a better solution to reduce boilerplate code, I'm all ears 😉

Reply:
Ah, that's a good point. I never tried that, so I trust what you say :)